src/sections/Meshery/Meshery-integrations/Individual-Integrations/relatedIntegration.js

```diff
@@ -91,11 +116,11 @@ const RelatedIntegration = ({ category }) => {
       <Slider {...settings}>
         {
           IntegrationList.map((item, index) => {
-            if (item.category === category && item.status != "InProgress"){
+            if (item.frontmatter.category === category && item.status != "InProgress"){
```

Comment:
```suggestion
            if (item.frontmatter.category === category){
```

Once you apply the filter mentioned in the above comment